graylog2-server/src/main/java/org/graylog2/rest/resources/system/responses/LookupTableCachePurgingNodeResponse.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog2.rest.resources.system.responses;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class LookupTableCachePurgingNodeResponse {
```

Comment:
We usually use auto-value based classes for these kind of objects. It would be great if you can switch this class over to an auto-value. Thanks!

Reply:
If we use what I proposed earlier, we wouldn't need a new Response class

Reply:
But does it really make sense to use `@AutoValue` for such simple objects like API Request/Response? There is no need in generating a `hashCode`, `equals` and `toString`.

Reply:
﻿Not relevant anymore: ﻿﻿https://github.com/Graylog2/graylog2-server/commit/2633ea14194847156939d7f1fb3381cb586364d6